File: lab9/omontero9.c

//Oliver Montero
//Lab 09 in C
//

#include <stdio.h>
#include <stdlib.h>
#include <math.h>
#include <time.h>

int main(void)
{
	int guess = 0, x = 3, y, i;
	srand(time(NULL));
	y = 1 + (rand() % 20);
	printf("Rand = %d\n", y);

	while(guess < 1 || guess > 20)
	{
		printf("Enter a number between 1 and 20:\n");
		scanf("%d", &guess);
		if(guess < 1 || guess > 20)
		{
			printf("Number out of range, choose between 1 and 20: \n");
		}
	}

	if (guess != y)
	{
		printf("You did not guess the right number. The random number was %d\n", y);
		if(guess < y)
		{
			printf("Your guess was lower than the number\n");
		}
		else
		{
			printf("Your guess was higher than the number\n");
		}
	}
	else
	{
		printf("Wow! You guessed the random number. It was %d\n", y);
		for (i = 0; i < y; i++)
		{
			printf("%d. Winner Winner!\n", i + 1);
		}
	}
	return EXIT_SUCCESS; 
}

File: lab9/omontero9.java
//Oliver Montero
//Lab 09 in Java

import java.util.Scanner;
import java.util.Random;

public class omontero9 
{
	public static void main(String[] args)
	{
		int randy = 0, guess = 0;
		Scanner kb = new Scanner(System.in);
		Random rand = new Random();
		randy = rand.nextInt(20) + 1;
		System.out.println(randy);

		while(guess < 1 || guess > 20)
		{
			System.out.println("Please choose a number between 1 and 20");
			guess = kb.nextInt();

			if(guess < 1 || guess > 20)
			{
			 	System.out.println("Number must be in the range of 1-20: ");
			}
		}

		if(guess != randy)
		{
			if(guess > randy)
			{
				System.out.println("You didn't guess the right number. Your guess was too high. The random number was " + randy);
			}
			else
			{
				System.out.println("You didn't guess the right number. Your guess was too low. The random number was " + randy);
			}
		}
		else
		{
			System.out.println("Wow! You guess the right number!");
			for (int i = 0; i < randy; i ++)
			{
				System.out.println(i + 1 +". Winner Winner!");
			}
		}
	}
}
